### PR TITLE
Manage the theme using a submodule

### PR DESCRIPTION
**Pull request by jcayzac**
It seems to be the way recommended by the [getting started](https://gohugo.io/getting-started/quick-start/#step-3-add-a-theme) guide, and avoid an issue where the theme just goes AWOL and Hugo starts complaining about missing layouts.

### CODE REVIEW

**Comment by NemoOudeis** [comment]
why does this remove `layouts/_default/_markup/render-link.html`? has that changed on the theme repo?

**Comment by NemoOudeis** [comment]
...magic happened...

File: README.md [hunk 1]
```diff
@@ -18,12 +18,17 @@
    Hugo Static Site Generator v0.78.2/extended darwin/amd64 BuildDate: unknown
    ```
 
-2. [Fork](https://github.com/rakutentech/engineering-blog/fork) & Clone this repository
-3. Run the local server with:
+2. [Fork](https://github.com/rakutentech/engineering-blog/fork)
+3. Clone your repository and initialize its submodules
+   ```
+   > git clone <YOUR GIT REPO CLONE URL>
```
Comment (NemoOudeis) [inline README.md:24]: Should we instruct to do `git clone --recurse-submodules` <URL>` instead of 2 separate commands?

Comment (jcayzac) [reply README.md:24]: Probably better yes

**Review by NemoOudeis — APPROVED** [review]
LGTM

**Review by boazy — APPROVED** [review]

**Pull request merged**